setup.py

```diff
@@ -34,7 +17,17 @@
             'templates/*/*.em',
         ],
     },
-    install_requires=install_requires,
+    install_requires=[
+        'catkin_pkg >= 0.4.3',
```

Comment:
I did a quick peek on catkin_pkg [0.4.3...1.0.0](https://github.com/ros-infrastructure/catkin_pkg/compare/0.4.3...1.0.0) to see if there was anything that would merit bumping the dependency. Realistically we depend on py3 support so most folks will be getting catkin_pkg 1.0.0 anyway.

I don't think we run low-water-mark CI anywhere for the infra packages.

Reply:
> I don't think we run low-water-mark CI anywhere for the infra packages

I believe this is correct. In fact, ros-infrastructure CI now targets HEAD for other infra packages.